app/controllers/AddressLookupController.scala

```diff
@@ -195,6 +196,15 @@ class AddressLookupController @Inject()(journeyRepository: JourneyRepository, ad
     }
   }
 
+  // GET /renewSession
+  def renewSession: Action[AnyContent] = Action { implicit req =>
+    Ok.sendFile(new File("conf/renewSession.jpg")).as("image/jpeg")
```

Comment:
What is this mysterious JPEG? What strange powers does it possess?

Reply:
It possesses no strange powers and is to ensure that we are returning some simple

Reply:
My point is "Why do we need to send a JPEG which we keep as a resource?" Why not just, say, a 1 space? What is the point of this?